integral_view/forms/zfs_forms.py

```diff
@@ -15,15 +15,33 @@ class BaseDatasetForm(CommonPropertiesForm):
     ch = [('G', 'GB'), ('M', 'MB')]
     quota_unit = forms.ChoiceField(choices=ch)
 
+
 class BaseZvolForm(CommonPropertiesForm):
 
     pass
 
+
 class CreateDatasetForm(BaseDatasetForm):
 
     pool = forms.CharField()
 
 
+class AdvancedDatasetZvolPropertiesForm(forms.Form):
+    name = forms.CharField()
+    property_value = forms.CharField()
+
+    def __init__(self, *args, **kwargs):
+        if kwargs and 'modifyable_properties' in kwargs:
```

Comment:
It did not look right but the internet was down so cld not check:) Updated it now.